rllib/evaluation/postprocessing.py

```diff
@@ -208,7 +208,7 @@ def compute_gae_for_sample_batch(
             #  This implementation right now will compute even the action_dist which
             #  will not be needed but takes time to compute.
             if policy.framework == "torch":
-                input_dict = convert_to_torch_tensor(input_dict)
+                input_dict = convert_to_torch_tensor(input_dict, device=policy.device)
```

Comment:
This prevents a bug where, when instantiating a policy on a rollout worker that has a GPU, we would fail because models are on GPU but input_dict is not.